### ISSUE TITLE
www.swift.org/sswg/incubated-packages reports "Not Found"

### ISSUE THREAD

**Issue by tkremenek** [swiftlang/swift-org-website#759]
The URL "https://www.swift.org/sswg/incubated-packages" resolves to "Not Found."

This link is referenced from:

https://www.swift.org/sswg/incubation-process.html

under "Resources and References"

**Comment by alexandersandberg** [comment]
Thanks, @tkremenek! I have opened a PR to fix this: #760

**Comment by daveverwer** [comment]
Now this is fixed, it’s a strange bug. Jekyll should absolutely be making those original links work, and as you note, it does for other pages.

The best way to create inter-page links with Jekyll is to use the [built in `link` function](https://jekyllrb.com/docs/liquid/tags/#links), like you can see here:

https://github.com/swiftlang/swift-org-website/blob/1bf7cf5ea9f8ebf99bea76389bd471ed54f7047e/packages/index.md?plain=1#L6C220-L6C294

The advantage of these is that if a page ever gets moved, they cause a build-time exception:

```
  Liquid Exception: Could not find document 'website-workgroup/index.md' in tag 'link'. Make sure the document exists and the path is correct. in blog-post-contributions/index.md
```

However, very few links in the site are currently referenced this way and it would be a big job to change them all.

Also, I just tried using this syntax on the first link fixed in the PR and including the link like this:

```
[here]({% link /sswg/incubation-process.md %} "SSWG Incubation Process")
```

Gives this error:

```
  Liquid Exception: Could not find document '/sswg/incubation-process.md' in tag 'link'. Make sure the document exists and the path is correct. in documentation/server/index.md
             Error: Could not find document '/sswg/incubation-process.md' in tag 'link'. Make sure the document exists and the path is correct. 
             Error: Run jekyll build --trace for more information.
```

So something isn’t working quite right. I don’t have more time now to look at this but I’m going to open an issue to look at this more closely.
